### PR TITLE
Deploy Mobile Workers

### PR DESCRIPTION
**Pull request by biyeun**
@dimagi/product 

Removes old Mobile Workers Views.
Makes New Mobile Workers View standard.

@benrudolph @esoergel

### CODE REVIEW

File: corehq/apps/hqwebapp/utils.py [hunk 1]
```diff
@@ -264,12 +260,6 @@ def sidebar_to_dropdown(sidebar_items, domain=None, current_url_name=None):
           'html': None,
           'is_divider': False,
           'is_header': False,
-          'title': u'New Mobile Worker',
-          'url': '/a/sravan-test/settings/users/commcare/add_commcare_account/'},
```
Comment (esoergel) [inline corehq/apps/hqwebapp/utils.py]: lol

Comment (esoergel) [inline corehq/apps/hqwebapp/utils.py]: @sravfeyn

**Comment by esoergel** [comment]
Is this ready for merge, or do you want the new version live for a couple days first?  I guess all the QA happened on staging, yeah?

File: corehq/apps/hqwebapp/utils.py (hunk 1 above)
Comment (biyeun) [inline corehq/apps/hqwebapp/utils.py]: omg lolllll

**Comment by biyeun** [comment]
@esoergel yep it's good to merge, all the qa happened on staging. the minor tweak of showing a loading spinner when searching can wait for an additional update, according to mealz

File: corehq/apps/hqwebapp/utils.py (hunk 1 above)
Comment (sravfeyn) [inline corehq/apps/hqwebapp/utils.py]: What!! There is one more on line 274. Did I add these? Will check and cleanup.

Comment (sravfeyn) [inline corehq/apps/hqwebapp/utils.py]: Wait, that's just a comment, no?

**Comment by czue** [comment]
kicked build - didn't look real.

**Comment by czue** [comment]
this is exciting!

File: corehq/apps/hqwebapp/utils.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/hqwebapp/utils.py]: Haha, you're absolutely right, @sravfeyn - I didn't zoom out the diff enough to see that.  Looks like this part should probably stay as-is, except maybe with the URL name and output URL updated (not that that's necessary either).

I'm not going to block this PR on that, but maybe @biyeun or @sravfeyn should revert that part in another PR.

**Comment by esoergel** [comment]
:balloon:

File: corehq/apps/users/views/__init__.py [hunk 2]
```diff
@@ -119,7 +119,8 @@ def redirect(self):
             user = CouchUser.get_by_user_id(self.couch_user._id, self.domain)
             if user:
                 if user.has_permission(self.domain, 'edit_commcare_users'):
-                    redirect = reverse("commcare_users", args=[self.domain])
+                    from corehq.apps.users.views.mobile import MobileWorkerListView
+                    redirect = reverse(MobileWorkerListView, args=[self.domain])
```
Comment (benrudolph) [inline corehq/apps/users/views/__init__.py]: @biyeun this should be `MobileWorkerListView.urlname`

**Pull request merged**